system/jupyter-cron-reports/templates/cornjob-monthly-ticket-analysis-report.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: "cron-monthly-ticket-analysis-report"
+  namespace: jupyterhub
+spec:
+  schedule: "0 0 1 * *"  # Schedule the job to run at 00:00 on the 1st day of every month
+  jobTemplate:
+    spec:
+      backoffLimit: 2
+      activeDeadlineSeconds: 1800
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+          - name: "cron-monthly-ticket-analysis-report"
+            image: "{{ .Values.global.registry }}/{{ .Values.jupyterhub_cron.images.application.name }}:{{ .Values.jupyterhub_cron.images.application.tag }}"
+            command: ['sh', '-c']
+            args:
+            - >
+              papermill /notebooks-git/Ticket_analysis_script.ipynb /notebooks-git/Ticket_analysis_output.ipynb
+            volumeMounts:
+            - name: notebooks-git
+              mountPath: /notebooks-git
+            env:
+            {{- if not (and $.Values.global.linkerd_enabled $.Values.global.linkerd_requested) }}
+            - name: LINKERD_AWAIT_DISABLED
+              value: "Linkerd was not enabled or not requested"
+            {{- end }}
+            - name: swift_url
+              valueFrom:
+                secretKeyRef:
+                  name: jupyter-cron-reports-secrets
+                  key: swift_url
+            - name: swift_password
+              valueFrom:
+                secretKeyRef:
+                  name: jupyter-cron-reports-secrets
+                  key: swift_password
+            - name: vault_token

```

Comment:
Here you are referencing a token that should be stored in the regional secrets file. `vault_token` is not a valid reference. Also if you want to run the notebook with a cronjob, I would recommend using the approle which is already part of the regional secrets file.

Reply:
Thank you. will update this